gax-grpc/src/main/java/com/google/longrunning/OperationsClient.java

```diff
@@ -519,6 +519,61 @@ private final void deleteOperation(DeleteOperationRequest request) {
     return stub.deleteOperationCallable();
   }
 
+  /**
+   * Waits for the specified long-running operation until it is done or reaches at most a specified
+   * timeout, returning the latest state. If the operation is already done, the latest state is
+   * immediately returned. If the timeout specified is greater than the default HTTP/RPC timeout,
+   * the HTTP/RPC timeout is used. If the server does not support this method, it returns
+   * `google.rpc.Code.UNIMPLEMENTED`. Note that this method is on a best-effort basis. It may return
+   * the latest state before the specified timeout (including immediately), meaning even an
+   * immediate response is no guarantee that the operation is done.
+   *
+   * <p>Sample code:
+   *
+   * <pre><code>
+   * try (OperationsClient operationsClient = OperationsClient.create()) {
+   *   String name = "";
+   *   WaitOperationRequest request = WaitOperationRequest.newBuilder()
+   *     .setName(name)
+   *     .setTimeout(Duration.ofMillis(100))
+   *     .build();
+   *   Operation response = operationsClient.waitOperation(request);
+   * }
+   * </code></pre>
+   *
+   * @param request The request object containing all of the parameters for the API call.
```

Comment:
The --> the
no period

Reply:
Done.

Reply:
need to push?

Reply:
Pushed, thanks.

Reply:
The --> the
no period
per Oracle JavaDoc guidelines